### ISSUE TITLE
Allow to upload but not write or delete

### ISSUE THREAD

**Issue by valerio-bozzolan** [mnutt/davros#127]
First of all, thank you for this project! It's very promising!

I don't understand if the current Davros Sandstorm app allows to share a link to upload stuff, without being able to delete other stuff.

Is this possible at the moment? If not, is this an interesting feature?

Thank you so much!

**Comment by ocdtrekkie** [comment]
I don't think Davros can currently do this, though I personally think there'd be a neat feature here to let it act more in a "dropbox" mode.

**Comment by mnutt** [comment]
This sounds like a neat feature. Perhaps integrated in with Sandstorm's capabilities.

**Comment by zenhack** [comment]
Yeah, this should be as simple as breaking out the existing write permission into finer grained permissions/roles, and updating the relevant checks.
